python/cairo-core/src/cairo_core/bytes.cairo

```diff
@@ -76,13 +76,8 @@ struct OptionalBytes {
     value: BytesStruct*,
 }
 
-struct StringStruct {
-    data: felt*,
-    len: felt,
-}
-struct String {
-    value: StringStruct*,
-}
+using String = Bytes;
```

Comment:
here there is a slight discrepancy, in RLP we say bytearray is of type Bytes and for String we say it's an alias

```
struct ExtendedEnum {
    sequence: SequenceExtended,
    bytearray: Bytes,
    bytes: Bytes,
    uint: Uint*,
    fixed_uint: Uint*,
    str: String,
    bool: Bool*,
}
```
then we should do using Bytearray = Bytes, no?

Reply:
yes, we could do the same for bytearray.